README.md

```diff
@@ -65,7 +61,7 @@ scala/
 
 ## Requirements
 
-You'll need a Java SDK (6 or newer), Apache Ant (version 1.9.0 or above), and curl (for `./pull-binary-libs.sh`).
+You'll need a Java SDK (6 for 2.11.x, 8 for 2.12.x), Apache Ant (version 1.9.0 or above), and curl (for `./pull-binary-libs.sh`).
```

Comment:
Ah, good catch. I've added an additional commit that fixes that, and also fixes the redundancy between the main readme and the Eclipse & IntelliJ readmes.